.circleci/config.yml

```diff
@@ -116,6 +116,7 @@ jobs:
       - run:
           name: Build
           command: |
+            cp .env frontends/election-manager/.env
```

Comment:
I feel like there must be a better solution to this problem but for some reason circleCI doesn't seem to be looking at the root-level .env file, which only matters for getting the ms-sems converter value in VxAdmin, this fixed it but I don't know why I need to do this.

Reply:
FWIW it looks like this file did exist but is deleted by this PR.

Reply:
yeah but it shouldn't be necessary